sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -239,6 +244,48 @@ public async Task CreateAndGetAzureDataLakeStorageGen2DataFeedWithOptionalMember
             ValidateAzureDataLakeStorageGen2DataSource(createdDataFeed.DataSource as AzureDataLakeStorageGen2DataFeedSource);
         }
 
+        [RecordedTest]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/21623")]
+        public async Task CreateAndGetAzureEventHubsDataFeedWithMinimumSetup()
```

Comment:
Two issues here (both tracked by https://github.com/Azure/azure-sdk-for-net/issues/21623).
- The service does not accept an Event Hubs data source with "mock" parameters, as we do for other data sources. We need to set up an actual Event Hubs resource to test this.
- I ran the tests locally and they pass except for one property (`IngestionStartTime`) that the service seems to be storing incorrectly. Double checked and it does not seem to be on client-side. I'll follow up with them.